### PR TITLE
Move to .md extension so the "See the changelog." link points to a ge…

### PR DESCRIPTION
**Pull request by hboetes**
…nerated html page.

Look at https://nicotine-plus.github.io/nicotine-plus/ and click the "See the changelog."

It downloads the NEWS file, whereas we'd rather have a nicely generated page. This simple PR aims to fix that.

The page is already mostly in .md format.

### CODE REVIEW

**Comment by mathiascode** [comment]
Also update the link in the README to point at NEWS.md.

**Comment by hboetes** [comment]
That rebasing is a tricky business. I hope I got it right this time.

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,1440 @@
+News
+====
+
+    As per GCS § 6.7, this file contains a list of user-visible noteworthy
+    changes. Note that this is not the same as a changelog.
+
+Version 2.0.0 (14 July 2020)
```
Comment (kiplingw) [inline NEWS.md:7]: This still isn't the latest news. Should be 2.0.1 from @mathiascode commit.

**Comment by hboetes** [comment]
OK, what am I missing here?
```
% git remote -v                        
origin	git@github.com:hboetes/nicotine-plus.git (fetch)
origin	git@github.com:hboetes/nicotine-plus.git (push)
vannelle	https://github.com/Nicotine-Plus/nicotine-plus.git (fetch)
vannelle	https://github.com/Nicotine-Plus/nicotine-plus.git (push)
% git fetch vannelle 
% git merge-base master vannelle/master       
5391714b45d3a27991aa080fb0fe138a9d9eedd4
% git rebase -i 5391714b45d3a27991aa080fb0fe138a9d9eedd4
Successfully rebased and updated refs/heads/master.                                                                                                                                                        
% git rebase vannelle/master                         
Current branch master is up to date.
% git push -f
Everything up-to-date
```

**Comment by mathiascode** [comment]
You have two files in your branch, one named NEWS with the 2.0.1 changes, and the NEWS.md file without the changes.

**Comment by hboetes** [comment]
Ah, that's odd. I did a git mv... How's that even possible?

**Comment by hboetes** [comment]
Great! Now we have: https://nicotine-plus.github.io/nicotine-plus/NEWS.html

**Pull request merged**